src/cameras/PerspectiveCamera.js

```diff
@@ -124,19 +124,19 @@ THREE.PerspectiveCamera.prototype.updateProjectionMatrix = function () {
 
 THREE.PerspectiveCamera.prototype.clone = function () {
 
-	var camera = new THREE.PerspectiveCamera();
+	var camera = new this.constructor( this.fov, this.aspect, this.near, this.far );
+	return camera.copy( this );
 
-	camera.copy( this );
-
-	camera.zoom = this.zoom;
-
-	camera.fov = this.fov;
-	camera.aspect = this.aspect;
-	camera.near = this.near;
-	camera.far = this.far;
-
-	return camera;
+};
 
+THREE.PerspectiveCamera.prototype.copy = function ( source ) {
+	
+	THREE.Camera.prototype.copy.call( this, source );
+	
+	this.zoom = source.zoom;
```

Comment:
Thanks for pointing me to the unit tests file. I was looking for that.

Looks like there's 21 failing (same number as in dev branch); all failing tests mention "Can't find variable: _" but it doesn't doesn't seem related to this PR's changes. Good to go, it seems.